pep-0703.rst

```diff
@@ -1531,6 +1531,52 @@ author is not aware of a way to add write barriers to CPython without
 substantially breaking the C-API.
 
 
+Why Not Deprecate ``PyDict_GetItem`` in Favor of ``PyDict_FetchItem``?
+----------------------------------------------------------------------
+
+This PEP proposes a new API ``PyDict_FetchItem`` which behaves like
+``PyDict_GetItem``, but returns a new reference instead of a borrowed
+reference.  As described in `Borrowed References`_, some uses of
```

Comment:
FYI, while it's likewise up to you as the PEP author, you might want to consider using [explicit ref target labels and the `:ref:` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref), i.e. `` :ref:`703-borrowed-refs` `` here and

```rst
.. _703-borrowed-refs:

Borrowed References
-------------------
```

above the section you want, instead of implicit hyperlinks. They auto-update rather than silently breaking (with no programmatic or visual indication) if you ever move or change the section name, warn if you make a typo or accidentally delete the label, work across files and even other docs sites, make it explicit that you're linking to an internal location instead of an external URL, and allow linking to any block of text, not just section headings.